octobot/src/server/github_handler.rs

```diff
@@ -513,6 +513,19 @@ impl GithubEventHandler {
         participants
     }
 
+    fn reviewer_names(&self, pull_request: &github::PullRequest) -> Vec<String> {
+        let mut reviewer_names = vec![];
+
+        if let Some(ref reviewers) = pull_request.requested_reviewers {
+            reviewer_names.extend(self.slack_user_names(reviewers));
+        }
+        if let Some(ref teams) = pull_request.requested_teams {
+            reviewer_names.extend(teams.into_iter().map(|t| format!("@{}", t.slug)));
```

Comment:
looking at GH api responses, looks like `slug` is just the name; does not contain an `@`.